src/App-au.vue

```diff
@@ -35,5 +31,15 @@ export default defineComponent({
       default: (props: AppProps) => props.lang,
     },
   },
+  computed: {
+    currentWidget() {
+      switch (this.theme) {
+        case ThemeEnum.Humm90:
+          return WidgetMainHumm90
+        default:
+          return WidgetMainHumm90
```

Comment:
is returning the same value something for the future or an accident? Might be worth adding some kind of `TODO` comment?

Reply:
Awesome! Thanks for noticing!

Reply:
Added a comment `TODO: add future AU variants below`